src/Files.App/Utils/Storage/Operations/FilesystemOperations.cs

```diff
@@ -465,7 +474,13 @@ public Task<IStorageHistory> DeleteAsync(IStorageItem source, IProgress<StatusCe
 
 		public async Task<IStorageHistory> DeleteAsync(IStorageItemWithPath source, IProgress<StatusCenterItemProgressModel> progress, bool permanently, CancellationToken cancellationToken)
 		{
-			StatusCenterItemProgressModel fsProgress = new(progress, true, FileSystemStatusCode.InProgress);
+			StatusCenterItemProgressModel fsProgress = new(
+				progress,
+				true,
+				FileSystemStatusCode.InProgress);
+
+			// TODO: Get total size
```

Comment:
I don't think we need the size for delete operations.

Reply:
Removed all